templates/upstart.conf.j2

```diff
@@ -7,4 +7,4 @@ respawn
 setuid {{ gunicorn_user }}
 chdir {{ gunicorn_app_dir }}
 
-exec gunicorn --config /etc/gunicorn/{{ gunicorn_app_name }}.py {{ gunicorn_wsgi }}
+exec gunicorn --config /etc/gunicorn/{{ gunicorn_app_name }}.py {{ gunicorn_wsgi }} {% if gunicorn_syslog %} 2>&1 | logger -t {{ gunicorn_app_name }} {% endif %}
```

Comment:
I didn't consider that, but sending to `logger` may still be more reliable. By default `gunicorn` sends to `udp://localhost:514` and `rsyslog`s udp and tcp listeners are off by default.